### DISCUSSION TITLE
这里提到的“去除模板中的block前后的空白字符和换行”选项是在哪里开启？

### DISCUSSION THREAD

**Discussion by irisfish531** [zhaohongxuan/obsidian-weread-plugin#308] [Q&A] [answered]
[去除模板中的block前后的空白字符和换行](https://github.com/zhaohongxuan/obsidian-weread-plugin/wiki/%E5%8E%BB%E9%99%A4%E6%A8%A1%E6%9D%BF%E4%B8%AD%E7%9A%84block%E5%89%8D%E5%90%8E%E7%9A%84%E7%A9%BA%E7%99%BD%E5%AD%97%E7%AC%A6%E5%92%8C%E6%8D%A2%E8%A1%8C)

**Comment by zhaohongxuan** [comment, marked as answer]
之前做的一个feature，功能都开发完了，到最后没发布，这个本来要默认功能的，只是项目前期没有考虑到，导致现在修改会有 break change，影响用户历史数据，因为这个功能上线之后没有更新模板的用户笔记都会被换行搞错乱，所以没有上线。
当然现在可以加个选项，让用户自己修改模板，但是想着对于用户来说学习成本太高了，索性最后没上线。
